versioned_docs/version-2.x/reference/environment-variables.mdx

```diff
@@ -263,6 +263,9 @@ keys** page from the menu.
   [the XO console](https://console.early-birds.io/) under `Search > API Keys`
 - `FRONT_COMMERCE_ATTRAQT_RECOMMENDATION_API_URL`:
   [Attraqt's recommendation API url](https://attraqt.gitbook.io/developer-documentation/xo-recommendations/using-the-recommendations-api)
+- `FRONT_COMMERCE_ATTRAQT_DISABLED` (*since 2.23*): setting it to `true` disables Attraqt search temporarily. This
```

Comment:
shouldn't it be also mentionned in https://developers.front-commerce.com/docs/magento2/search-engine#front-commerce-configuration-2 ?

Reply:
I wasn't sure were to put it as it's not something a developer might use when setting up the project nor in a standard use case. I wanted to avoid adding extraneous information there.

But if you see a good way to add it there (maybe an information callout?) I can add it

Reply:
a short note saying this variable/feature exists with a link to its description would be enough

Reply:
Commit amended to add a "Tip" admonition on https://deploy-preview-626--heuristic-almeida-1a1f35.netlify.app/docs/2.x/magento2/search-engine#front-commerce-configuration-3

![2023-04-13_11-42](https://user-images.githubusercontent.com/75968/231720987-36144053-aae9-44b3-8210-116b12f017b3.png)